install/k8s/contiv/contiv.yaml

```diff
@@ -262,52 +357,142 @@ spec:
                 configMapKeyRef:
                   name: contiv-config
                   key: contiv_netmode
-          securityContext:
-            privileged: true
           volumeMounts:
-            - mountPath: /etc/openvswitch
-              name: etc-openvswitch
-              readOnly: false
-            - mountPath: /lib/modules
-              name: lib-modules
-              readOnly: false
-            - mountPath: /var/run
-              name: var-run
+            - mountPath: /var/contiv
+              name: var-contiv
               readOnly: false
             - mountPath: /var/log/contiv
               name: var-log-contiv
               readOnly: false
+
+        - name: contiv-api-proxy
+          image: contiv/auth_proxy:1.2.0
+          args:
+            - --tls-key-file=/var/contiv/auth_proxy_key.pem
+            - --tls-certificate=/var/contiv/auth_proxy_cert.pem
+            - --data-store-address=$(STORE_URL)
+            - --data-store-driver=$(STORE_DRIVER)
+            - --netmaster-address=localhost:9999
+          env:
+            - name: NO_NETMASTER_STARTUP_CHECK
```

Comment:
We shouldn't have this env var set by default... this allows auth_proxy to send requests to an incompatible netmaster